scripts/remove-dependencies.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
Kannattaa laittaa `-euo pipefail` vaan kaikkiin skripteihin, niin skripti hajoaa myös määrittelemättömiin muuttujiin (-u) ja putket ei piilota exit codeja (-o pipefail -- tää on bashismi eli ei toimi sh:ssa).

eli `-o pipefail` tekee tän:

```
$ false | true
$ echo $?
0
$ set -o pipefail
$ false | true
$ echo $?
1
```

periaatteessa `scriprts/merge-metadata.sh`:ssa ois hyvä olla myös näin, mutta kun sen set-komentoa ei editoitu tässä muutoksessa, niin jätin (erillisen) kommentin laittamatta